internal/flink/command_artifact_create.go

```diff
@@ -14,6 +15,9 @@ import (
 	"github.com/confluentinc/cli/v3/pkg/utils"
 )
 
+
+var ALLOWED_RUNTIME_LANG = []string{"python", "java"}
```

Comment:
This formatting is not allowed in Go.
```suggestion
var allowedRuntimeLanguages = []string{"python", "java"}
```

Reply:
Thanks @brianstrauch  updated. Also i might add more code in this PR, let me know if it's easier for you to review once I open the PR.